src/actions/userActions.js

```diff
@@ -0,0 +1,52 @@
+import * as types from "../constants/ActionTypes";
+
+const googleLoginRef = "apiv1/googlelogins";
+
+export function toggleDarkMode(){
+    return{ type: types.TOGGLE_DARK_MODE };
+}
```

Comment:
Isn't this @bryanmontalvan's Dark mode?

If you're relying on his work for a user profile, maybe we should split those changes out such that you don't need to add his code before it's in a finished state